packages/components/inputs/localized-rich-text-input/src/editor.js

```diff
@@ -75,7 +81,12 @@ const Editor = (props) => {
     onToggle();
   }
 
-  const languagesControl = props.languagesControl();
+  const shouldToggleButtonTakeSpace =
+    // if hasLanguagesControl and there are no errors/warnings to display
+    // then the toggleButton is absolutely positioned
```

Comment:
This is because the toggle button is placed next to the LanguagesControl (now called LocalizedInputToggle) without being siblings in the DOM.

If there is a error or warning showing, then it can be placed statically because it will then be a sibling to the error/warning message, with the LocalizedInputToggle being pushed down.